operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/informer/Field.java

```diff
@@ -0,0 +1,10 @@
+package io.javaoperatorsdk.operator.api.config.informer;
+
+public @interface Field {
+
+  String path();
+
+  String value();
+
+  boolean negate() default false;
```

Comment:
I would use `negated` here as well to be consistent (and I think it makes more sense that way)